Dockerfile.staging

```diff
@@ -20,7 +20,7 @@ RUN ["sh", "-c", "mv .github/workflows/staging/openmetrics.d/${chain}-conf.yaml
 RUN service datadog-agent start
 
 RUN go get ./...
-RUN go build github.com/ChainSafe/gossamer/cmd/gossamer
+RUN go install -trimpath github.com/ChainSafe/gossamer/cmd/gossamer
```

Comment:
That will build from the master branch from github.com, I don't think that's what we want?
Just add `-o gossamer` to the `go build` command instead?

Reply:
I validated this works.  I think it's cause we are currently in the same directory as `go.mod` which references this module as `github.com/ChainSafe/gossamer`

Reply:
Interesting, TIL, sorry about the wrong assumption!